index.bs

```diff
@@ -8960,24 +8963,179 @@ The [=remote end steps=] given |session|, and |command parameters| are:
 
 </div>
 
+#### The input.interceptFileDialogs Command ####  {#command-input-interceptFileDialogs}
+
+The <dfn export for=commands>input.interceptFileDialogs</dfn> command
```

Comment:
should we perhaps intercept by the fact of the event subscription instead of the dedicated method? it probably makes sense to have interception always enabled otherwise the dialog will block the execution?